frontend/src/components/pages/rp-connect/secrets/Secrets.QuickAdd.tsx

```diff
@@ -0,0 +1,190 @@
+import {
+  Button,
+  Flex,
+  FormField,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  PasswordInput,
+  Select,
+  Text,
+  createStandaloneToast,
+  isSingleValue,
+  useDisclosure,
+} from '@redpanda-data/ui';
+import { useState } from 'react';
+import {
+  CreateSecretRequest,
+  Scope,
+  type Secret,
+} from '../../../../protogen/redpanda/api/dataplane/v1alpha2/secret_pb';
+import { rpcnSecretManagerApi } from '../../../../state/backendApi';
+import { base64ToUInt8Array, encodeBase64 } from '../../../../utils/utils';
+import { formatPipelineError } from '../errors';
+
+const { toast } = createStandaloneToast();
+
+type SecretsQuickAddProps = {
+  isOpen: boolean;
+  onCloseAddSecret: () => void;
+  onAdd: (secretId: string) => void;
+};
+
+const SecretsQuickAdd = ({ isOpen, onAdd, onCloseAddSecret }: SecretsQuickAddProps) => {
+  const [searchValue, setSearchValue] = useState('');
+  const [secret, setSecret] = useState('');
+  const [id, setId] = useState('');
+  const [isCreating, setIsCreating] = useState(false);
+  const { isOpen: isNewSecret, onClose: disableNewSecret, onOpen: enableNewSecret } = useDisclosure();
+
+  const availableSecrets =
+    rpcnSecretManagerApi.secrets?.map((s) => ({
+      label: s.id,
+      value: s,
+    })) ?? [];
+
+  const addSecret = async (id: string) => {
+    id = id.toUpperCase();
+    if (isNewSecret) {
+      setIsCreating(true);
+      const result = await rpcnSecretManagerApi
+        .create(
+          new CreateSecretRequest({
+            id: id,
+            // @ts-ignore js-base64 does not play nice with TypeScript 5: Type 'Uint8Array<ArrayBufferLike>' is not assignable to type 'Uint8Array<ArrayBuffer>'.
+            secretData: base64ToUInt8Array(encodeBase64(secret)),
```

Comment:
I left this comment in another component, can you please check if it continues to fail or maybe we can get rid of the ignore?

Reply:
yes it continue failing